Library/Homebrew/brew.sh

```diff
@@ -4,6 +4,11 @@ then
   export LC_ALL="en_US.UTF-8"
 fi
 
+# The USER env var is not guaranteed to be available, so fall back to id if not
+# provided. We destructively set it in the environment to make it available to
```

Comment:
I suggest removing the word `destructively`. It seems a bit misleading to me, since it's only set to `id -un` if it's not already provided by the environment. I'm otherwise :+1: on this PR.

Reply:
Good call.